gandalf.js

```diff
@@ -1,117 +1,121 @@
-const formattedDate = new Date().toLocaleDateString();
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.